### PR TITLE
PRO-1582 normalize widget chrome

### PR DESCRIPTION
**Pull request by stuartromanek**
- widgets at all depths have the same visual look
- breadcrumbs are now theme inverted along with widget controls
- secondary theme color removed

### CODE REVIEW

File: modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue [hunk 1]
```diff
@@ -615,11 +605,8 @@ export default {
     align-items: center;
     margin: 0;
     padding: 2px;
-    background-color: var(--a-primary);
-  }
-
-  .apos-area-widget-inner .apos-area-widget-inner .apos-area-widget__breadcrumbs {
-    background-color: var(--a-secondary);
+    background-color: var(--a-background-primary);
+    border: 1px solid var(--a-primary-button-hover);
```
Comment (stuartromanek) [inline modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue:609]: @abea i know this un-semantic use of variables will grind your gears, I suggest a ticket that converts them to something more agnostic of use, more like the base vars

Comment (abea) [reply modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue:609]: Is this not a find all and replace quick thing? Why not do it in this change?

Comment (stuartromanek) [reply modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue:609]: because it seemed like a different thing that would grind your gears

Comment (stuartromanek) [reply modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue:609]: i can do it that way though, sure

Comment (abea) [reply modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue:609]: Yeah, it'd be better to get it done. No idea when the ticket would happen.

**Review by abea — COMMENTED** [review]
See comment on the var names.

**Review by abea — APPROVED** [review]

**Pull request merged**